docs/appendices/release-notes/unreleased.rst

```diff
@@ -57,6 +57,10 @@ None
 Fixes
 =====
 
+- Fixed an issue that would lead to incorrect result when selecting the
+  :ref:`cluster license <sys-cluster-license>` object column, namely, the
+  fields of the object would contain the null values, even though the license.
```

Comment:
I think the the sentence is not complete :) 

```suggestion
  fields of the object would contain the null values, even though the license existed.
```

Reply:
oh no :/